src/components/shared/MarkerTooltipContents.js

```diff
@@ -40,9 +40,14 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
         );
       }
       case 'DOMEvent': {
+        let latency;
+        if (data.timeStamp) {
+          latency = `${formatTimeLength(data.startTime - data.timeStamp)}ms`;
+        }
```

Comment:
I agree but the difference is almost impossible to be zero because we use high-res timestamp.